src/coreComponents/fileIO/Outputs/OutputBase.cpp

```diff
@@ -64,14 +62,33 @@ void OutputBase::initializePreSubGroups()
   // SetupDirectoryStructure();
 }
 
+
+
+string const & OutputBase::getOutputDirectory()
+{
+  static string m_outputDirectory;
+  return m_outputDirectory;
+}
+
 void OutputBase::setOutputDirectory( string const & outputDir )
 {
-  m_outputDirectory = outputDir;
+  string & outputDirectory = const_cast< string & >( getOutputDirectory() );
+  outputDirectory = outputDir;
+  FunctionBase::setOutputDirectory( outputDirectory );
```

Comment:
@MelReyCG I did this instead of moving the `outputDirectory` up to `common`. We will have to come back and do something better later, but I didn't like the move option either....maybe we can discuss at the next meeting.